configure.ac

```diff
@@ -2232,7 +2233,7 @@ fi
 dnl ============================================================================
 dnl Enable background threads if possible.
 
-if test "x${have_pthread}" = "x1" -a "x${je_cv_os_unfair_lock}" != "xyes" ; then
+if test "x${have_pthread}" = "x1" -a "x${je_cv_os_unfair_lock}" != "xyes" -a "x${is_cywgin}" = "x1" ; then
```

Comment:
I don't think this is the right test -- it will effectively disable background threads on non-cygwin configurations, right?

Reply:
I think you are correct. I will check and correct that. Thanks.